client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -1,82 +1,69 @@
 import SignupForm from './signup-form';
-import SiteTitle from './site-title';
 import CreateSite from './create-site';
 import { Attributes } from './types';
-import { Step } from '../steps';
+import { Step, usePath } from '../path';
 import './style.scss';
 import VerticalBackground from './vertical-background';
-import Link from '../components/link';
+import AcquireIntent from './acquire-intent';
 
 const OnboardingEdit: FunctionComponent< BlockEditProps< Attributes > > = () => {
-	const { siteVertical, siteTitle, selectedDesign } = useSelect( select =>
+	const { siteVertical, selectedDesign } = useSelect( select =>
 		select( STORE_KEY ).getState()
 	);
 	const isCreatingSite = useSelect( select => select( SITE_STORE ).isFetchingSite() );
 
+	const makePath = usePath();
+
 	return (
 		<>
 			<VerticalBackground />
-			{ isCreatingSite && <Redirect push to={ Step.CreateSite } /> }
+			{ isCreatingSite && <Redirect push to={ makePath( Step.CreateSite ) } /> }
 			<Switch>
-				<Route exact path={ Step.IntentGathering }>
-					<div className="onboarding-block__acquire-intent">
-						<div className="onboarding-block__questions">
-							<h2 className="onboarding-block__questions-heading">
-								{ ! siteVertical &&
-									! siteTitle &&
-									NO__( "Let's set up your website – it takes only a moment." ) }
-							</h2>
-							<StepperWizard
-								stepComponents={ [ VerticalSelect, ( siteVertical || siteTitle ) && SiteTitle ] }
-							/>
-							{ siteVertical && (
-								<div className="onboarding-block__footer">
-									<Link
-										to={ Step.DesignSelection }
-										className="onboarding-block__question-skip"
-										isLink
-									>
-										{ /* @TODO: add transitions and correct action */ }
-										{ siteTitle ? NO__( 'Continue' ) : NO__( "Don't know yet" ) } →
-									</Link>
-								</div>
-							) }
-						</div>
-					</div>
+				<Route path={ makePath( Step.IntentGathering ) }>
+					<AcquireIntent />
 				</Route>
-				<Route exact path={ Step.DesignSelection }>
-					{ ! siteVertical ? <Redirect to={ Step.IntentGathering } /> : <DesignSelector /> }
+
+				<Route path={ makePath( Step.DesignSelection ) }>
+					{ ! siteVertical ? (
+						<Redirect to={ makePath( Step.IntentGathering ) } />
+					) : (
+						<DesignSelector />
+					) }
 				</Route>
-				<Route exact path={ Step.PageSelection }>
+
+				<Route path={ makePath( Step.PageSelection ) }>
 					{ ! selectedDesign ? (
-						<Redirect to={ Step.DesignSelection } />
+						<Redirect to={ makePath( Step.DesignSelection ) } />
 					) : (
-						<DesignSelector showPageSelector={ true } />
+						<DesignSelector showPageSelector />
 					) }
 				</Route>
-				<Route exact path={ Step.Signup }>
-					<SignupForm />
+
+				<Route path={ makePath( Step.Signup ) }>
+					<SignupForm />;
 				</Route>
-				<Route exact path={ Step.CreateSite }>
+
+				<Route exact path={ makePath( Step.CreateSite ) }>
```

Comment:
All the other routes are no longer `exact`, so for consistency's sake, we should also remove the `exact` from this one:

```suggestion
				<Route path={ makePath( Step.CreateSite ) }>
```

Reply:
Fixed. https://github.com/Automattic/wp-calypso/pull/39312/commits/9dfe2fbfff8729ef60edbe6b9d971ad8a4054301